### ISSUE TITLE
[Select] Start the search when user starts typing

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#285]
If the select provides a search and is currently under the keyboard focus, the user has to enter space/enter to start the search. The user should be able to start the search when it starts typing as well.
The first key pressed, should be entered manually into the search input.
